pkgs/desktops/plasma-5/plasma-desktop/default.nix

```diff
@@ -122,19 +129,29 @@ mkDerivation {
     ./kcm-access.patch
   ];
   CXXFLAGS =
-    let
-      # run gsettings with desktop schemas for using in kcm_accces kcm
-      gsettings-wrapper = runCommandLocal "gsettings-wrapper" { nativeBuildInputs = [ makeWrapper ]; } ''
-        makeWrapper ${glib}/bin/gsettings $out --prefix XDG_DATA_DIRS : ${gsettings-desktop-schemas.out}/share/gsettings-schemas/${gsettings-desktop-schemas.name}
-      '';
-    in
     [
       ''-DNIXPKGS_HWCLOCK=\"${lib.getBin util-linux}/bin/hwclock\"''
-      ''-DNIXPKGS_GSETTINGS=\"${gsettings-wrapper}\"''
+      ''-DNIXPKGS_GSETTINGS=\"${gsettings-wrapper}/bin/gsettings\"''
     ];
   postInstall = ''
     # Display ~/Desktop contents on the desktop by default.
     sed -i "''${!outputBin}/share/plasma/shells/org.kde.plasma.desktop/contents/defaults" \
         -e 's/Containment=org.kde.desktopcontainment/Containment=org.kde.plasma.folder/'
   '';
+
+  dontWrapQtApps = true;
+  postFixup = ''
+    for program in $out/libexec/*; do
+      wrapQtApp "$program"
+    done
+
+    for program in $out/bin/*; do
```

Comment:
Should we just add it to PATH for all the binaries? This will simplify the package quite a bit and have no real cost as the reference is there already.

Reply:
i was doing like that to prevent system settings from wrapped with wrapped gsetting because it affects `kcm_access` when used on system setting

and this could have caused to miss regression on `kcm_access` gsettings patch

but this isnt big deal because we can test in `kcmshell` too. so now i changed it to use `qtWrapperArgs`